### PR TITLE
Implement Sign Verify SDK changes. 

### PR DESCRIPTION
**Pull request by SethBe**
This change implements the SDK changes described in https://github.com/microsoft/wssdagent/pull/663

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -19,5 +19,6 @@ replace (
 	github.com/Azure/go-autorest v11.1.2+incompatible => github.com/Azure/go-autorest/autorest v0.10.0
 	github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.1.0
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
+	github.com/microsoft/moc => /home/sethbe/goroot/src/github.com/microsoft/moc //Sethbe
```
Comment (SethBe) [inline go.mod:22]: >github.com/microsoft/moc => /home/sethbe/goroot/src/githu [](http://example.com/codeflow?start=1&length=57)

fix before final commit

File: services/security/keyvault/keyvault.go [hunk 2]
```diff
@@ -235,6 +235,50 @@ type KeyImportExportValue struct {
...
+	Digest *string `json:"digest,omitempty"`
+
+	// Value - a URL-encoded base64 string of the signature to be verifed
+	Signature *string `json:"value,omitempty"`
+}
+
+// KeyVerifyResult the key verify operation result.
+//https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.keyvault.models.keyverifyresult?view=azure-dotnet-legacy
+type KeyVerifyResult struct {
+	autorest.Response `json:"-"` //Sethbe what is this???do we need it
```
Comment (SethBe) [inline services/security/keyvault/keyvault.go:278]: >autorest.Response `json:"-"` //Sethbe what is this???do we need it [](http://example.com/codeflow?start=1&length=66)

Understand and fix prior to final commit.

File: services/security/keyvault/key/wssd.go [hunk 3]
```diff
@@ -361,6 +361,34 @@ func (c *client) UnwrapKey(ctx context.Context, group, vaultName, name string, p
...
+	result, err = getDataFromResponse(response)
+	return
+}
+
+func (c *client) Verify(ctx context.Context, group, vaultName, name string, param *keyvault.KeyVerifyParameters) (result *keyvault.KeyVerifyResult, err error) {
+	request, err := c.getKeyOperationRequestVerify(ctx, group, vaultName, name, param, wssdcloudcommon.KeyOperation_VERIFY)
+	if err != nil {
+		return
+	}
+	fmt.Println("Verify in wssd.go")
```
Comment (SethBe) [inline services/security/keyvault/key/wssd.go:382]: >fmt.Println("Verify in wssd.go") [](http://example.com/codeflow?start=1&length=32)

remove

File: services/security/keyvault/keyvault.go [hunk 4]
```diff
@@ -235,6 +235,50 @@ type KeyImportExportValue struct {
...
+	// Value - a URL-encoded base64 string of the digest used for signing
+	Digest *string `json:"digest,omitempty"`
+
+	// Value - a URL-encoded base64 string of the signature to be verifed
+	Signature *string `json:"value,omitempty"`
+}
+
+// KeyVerifyResult the key verify operation result.
+//https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.keyvault.models.keyverifyresult?view=azure-dotnet-legacy
+type KeyVerifyResult struct {
```
Comment (raghavendra-nataraj) [inline services/security/keyvault/keyvault.go:277]: I believe this is stale, can we remove this

Comment (SethBe) [reply services/security/keyvault/keyvault.go:277]: Its actually not stale, This is the structure returned to the user as part of the sdk

Comment (SethBe) [reply services/security/keyvault/keyvault.go:277]: Checkout 
https://docs.microsoft.com/en-us/rest/api/keyvault/keys/verify/verify?tabs=HTTP#keyverifyresult

**Review by raghavendra-nataraj — APPROVED** [review]

**Pull request merged**